### PR TITLE
Release Notes for 3.43.0 (Java SDK)

### PR DESCRIPTION
**Pull request by Emdee89**
## What Has Changed?
Introducing release notes for SAP Cloud SDK for Java 3.43.0-

## Manual Checks?

- [x] Text adheres to the style guide (`vale docs/`)
  - Every sentence is on its own line
  - Headings use [title capitalization](https://capitalizemytitle.com/style/AP/#) (applies also to the sidebar and title of a document)
  - You followed [naming center](https://www.sapbrandtools.com/naming-center/#/dashboard) guidelines when referring to SAP products (e.g. SAP S/4HANA)
- [x] You checked your spelling and grammar (consider using Grammarly, see `CONTRIBUTING.md`)
- [x] You formatted all changed files with prettier (`npm run prettier`)
- [x] You tested if the documentation still builds (`npm run build`)
- [x] You verified all new and changed links still work (changing the `id` or name of a file can break links)
- [x] You have updated the [feature matrix](https://github.com/SAP/cloud-sdk/blob/main/docs/components/data/features.js) if you add documentation on a new feature or otherwise required.

### CODE REVIEW

**Review by Johannes-Schneider — APPROVED** [review]
LGTM.

File: docs/java/release-notes.mdx [hunk 1]
```diff
@@ -54,6 +54,42 @@ To continue using the latest features outlined in the release notes for version
 ### Fixed issues
 -->
 
+## 3.43.0 - April 22, 2021
+
+[Maven Central](https://search.maven.org/search?q=g:com.sap.cloud.sdk*%20AND%20v:3.43.0) | [Javadoc](https://help.sap.com/doc/55bb1642ea614bf582d514ed34401136/1.0/en-US/index.html)
+
+### New Functionality
+
+- We added a new API for creating OData V2 Changesets in Batch requests that mimics the existing OData V4 API.
```
Comment (artemkovalyov) [inline docs/java/release-notes.mdx:63]:
```suggestion
- We added a new API for creating OData v2 Changesets in Batch requests that mimics the existing OData V4 API.
```

File: docs/java/release-notes.mdx [hunk 2]
```diff
@@ -54,6 +54,42 @@ To continue using the latest features outlined in the release notes for version
...
+  This new API also supports custom headers on a per-request basis.
```
Comment (artemkovalyov) [inline docs/java/release-notes.mdx:64]:
```suggestion
  As a natural outcome, this new API also supports custom headers on a per-request basis.
```

**Review by artemkovalyov — APPROVED** [review]
LGTM. Check minor suggestions.

**Pull request merged**